src/index.js

```diff
@@ -151,6 +152,12 @@ const Index = () => {
                                                 component={userIsAuthenticated(EditMetadataItem)}
                                             />
                                         </Route>
+                                        <Route path={`versions/:versionID/cantabular`} component={config.enableCantabularJourney ? userIsAuthenticated(CantabularMetadataController) : null}>
```

Comment:
Is it worth having the flag here? If it is off we won't access this route and because this is only an internal system I wouldn't control this with the feature flag so to reduce the changes we'll need to make once we remove it? But there is nothing wrong with this if you prefer to keep it.

Reply:
I've removed the flag to minimise the changes.